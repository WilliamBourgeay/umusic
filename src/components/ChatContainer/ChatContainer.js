// @flow

import * as React from 'react';
import Paper from '@material-ui/core/Paper';

import './ChatContainer.css';

type Props = {
  children: React.Node,
  className?: string,
};

const ChatContainer = ({ children, className }: Props) => (
  <Paper className={`ChatContainer ${className || ''}`} elevation={3}>
    {children}
  </Paper>
);

ChatContainer.defaultProps = {
  className: '',
};

export default ChatContainer;
