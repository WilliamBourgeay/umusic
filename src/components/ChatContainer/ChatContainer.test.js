import * as React from 'react';
import renderer from 'react-test-renderer';

import ChatContainer from './ChatContainer';

describe('ChatContainer', () => {
  it('should render a ChatContainer', () => {
    const tree = renderer.create(
      <ChatContainer>
        <div />
      </ChatContainer>,
    );

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
