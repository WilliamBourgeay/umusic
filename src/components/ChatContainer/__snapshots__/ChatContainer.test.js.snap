// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatContainer should render a ChatContainer 1`] = `
<div
  className="MuiPaper-root-1 MuiPaper-elevation3-6 MuiPaper-rounded-2 ChatContainer "
>
  <div />
</div>
`;
